docs/api/cameras/Camera.html

```diff
@@ -34,6 +34,13 @@
 
 
 		<h2>Methods</h2>
+		
+		<h3>[method:null getWorldDirection]()</h3>
+		<div>
+		<br />
+		This return the world direction.

```

Comment:
Returns a vector representing the direction in which the camera is looking, in world space.